cmd/version.go

```diff
@@ -17,7 +20,7 @@ func newVersionCmd() *cobra.Command {
 		Long:  "Print the version of ACS-Engine",
 
 		Run: func(cmd *cobra.Command, args []string) {
-			log.Infof("ACS-Engine Version: %s", BuildSHA)
+			log.Infof("ACS-Engine Version: %s (%s)", BuildTag, BuildSHA)
```

Comment:
Happy to break that out into a separate piece of work if you can't get to it RN. I think we should just called out the `BuildTag:` and `BuildSHA` explicitly in the string so that we can interpret `unset`